### PR TITLE
Tag component changes

### PR DESCRIPTION
**Pull request by mainframev**
BREAKING CHANGE
- Icon prop removed
- Different colors
- Remove focusing if onClick/onRemove not provided
- rewrote to testing-library

3 commits in the end

I hope i got it right about colors, plz check if it's correct tokens @vepor 
<br/><br/><br/><url>LiveURL: https://orbit-feat-tag.surge.sh</url>

### CODE REVIEW

**Review by silvenon — CHANGES_REQUESTED** [review]
Otherwise looks good, and stories work fine. 👍

File: packages/orbit-components/src/Tag/index.js [hunk 1]
```diff
@@ -19,103 +20,106 @@ const getFontSize = ({ theme, size }) => {
...
+    padding: ${rtlSpacing(theme.orbit.paddingTag)};
+    transition: color ${theme.orbit.durationFast} ease-in-out,
+      box-shadow ${theme.orbit.durationFast} ease-in-out,
+      background ${theme.orbit.durationFast} ease-in-out;
+
+    &:focus {
+      outline: 0;
+    }
+
+    ${({ actionable }) =>
```
Comment (vepor) [inline packages/orbit-components/src/Tag/index.js:92]: Could be moved to the first function in the top - if we want to go with one function 🙂

File: packages/orbit-components/src/Tag/index.js [hunk 2]
```diff
@@ -153,35 +157,34 @@ const Tag = (props: Props) => {
...
-      icon={!!icon}
-      tabIndex="0"
+      tabIndex={(onClick || onRemove) && "0"}
       role="button"
-      onKeyDown={ev => buttonClickEmulation(ev, onClick)}
+      onKeyDown={buttonClickEmulation(onClick)}
     >
+      {children}
       {!!onRemove && (
         <CloseContainer
```
Comment (vepor) [inline packages/orbit-components/src/Tag/index.js:166]: The `CloseContainer` component doesn't have any transition 😕

**Review by vepor — CHANGES_REQUESTED** [review]
One last thing to resolve - the states of colors. Put it into the original thread on Slack.

File: packages/orbit-components/src/Tag/__tests__/index.test.js [hunk 3]
```diff
@@ -1,42 +1,29 @@
...
-
-  it("should have passed props", () => {
-    expect(component.render().prop("data-test")).toBe(dataTest);
-  });
-  it("should contain a content", () => {
-    expect(component.render().text()).toBe(content);
-  });
-  it("should contain a icon", () => {
-    expect(component.find("Bus").exists()).toBe(true);
+    userEvent.click(screen.getByLabelText("close"));
```
Comment (silvenon) [inline packages/orbit-components/src/Tag/__tests__/index.test.js:25]:
```suggestion
    userEvent.click(screen.getByRole("button", { name: "close" }));
```

**Review by silvenon — CHANGES_REQUESTED** [review]
Other than this minor suggestion everything else looks good. Regarding colors I'm not in the loop (not sure where that Slack thread is), so I'll leave it up to @vepor to confirm.

**Review by silvenon — APPROVED** [review]

**Review by silvenon — APPROVED** [review]
Looks good, and the close button has transitions now 👍

**Pull request merged**